Mathlib/RingTheory/Morita/Matrix.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2025 Yunzhou Xie. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yunzhou Xie, Jujian Zhang
+-/
+import Mathlib.Algebra.Category.ModuleCat.Basic
+import Mathlib.LinearAlgebra.Matrix.Module
+
+/-!
+## Main definitions
+- `ModuleCat.toMatrixModCat`: The functor from `Mod-R` to `Mod-Mₙ(R)` induced by
+  `LinearMap.mapModule` and `Matrix.matrixModule`.
+
+# TODO
+- Prove `Mod-R` and `Mod-Mₙ(R)` are morita-equivalent.
```

Comment:
Either `R` and `Mₙ(R)` are Morita-equivalent or `Mod-R` and `Mod-Mₙ(R)` are equivalent.